SparseGrids/tsgLinearSolvers.cpp

```diff
@@ -167,6 +167,69 @@ void TasmanianTridiagonalSolver::decompose(int n, double d[], double e[], double
     }
 }
 
+void TasmanianFourierTransform::dft(const int rank, const int n[], const std::complex<double> data[], std::complex<double> out[], const int direction){
```

Comment:
* rename the `dft` function to something like: `discrete_fourier_transform`

if you google `dft code` the top hits are related to Density Field Theory, which is a huge field. Nobody is really using the simple Fourier transfrom, and we will drop it soon.